hooks/batch/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"context"
+	"log/slog"
+	"strconv"
+
+	_ "hook/https"
+
+	"github.com/deckhouse/module-sdk/pkg"
+	"github.com/deckhouse/module-sdk/pkg/app"
+	objectpatch "github.com/deckhouse/module-sdk/pkg/object-patch"
+	"github.com/deckhouse/module-sdk/pkg/registry"
+)
+
+const (
+	SnapshotKey = "golang_versions"
+)
+
+var _ = registry.RegisterFunc(config, HandlerHook)
+
+// # Since we subscribed to ApiVersion example.io/v1, we get .spec.version (see jqFilter) as an
```

Comment:
fix commentaries, they must be in golang style (without #)
```suggestion
// Since we subscribed to ApiVersion example.io/v1, we get .spec.version (see jqFilter) as an
```